app/js/controllers.js

```diff
@@ -1797,8 +1797,31 @@ angular.module('myApp.controllers', ['myApp.i18n'])
         })
       }
       if (selectedMessageIDs.length) {
-        ErrorService.confirm({type: 'MESSAGES_DELETE', count: selectedMessageIDs.length}).then(function () {
-          AppMessagesManager.deleteMessages(selectedMessageIDs).then(function () {
+        var peerString = $scope.curDialog.peer
```

Comment:
Please move this code to `AppMessagesManager.canMessageBeRevoked` service method.
Might be a good idea to look at `AppMessagesManager.canMessageBeEdited` first.